src/utils/utils_concurrency.h

```diff
@@ -87,6 +87,15 @@ int utils_write_unlock(utils_rwlock_t *rwlock);
 
 void utils_init_once(UTIL_ONCE_FLAG *flag, void (*onceCb)(void));
 
+#define utils_atomic_store_u64(ptr, val)                                       \
```

Comment:
Is it needed? I think you can use `__atomic_store_n` or `__atomic_stor`

Reply:
this is for win so InterlockedExchange64

Reply:
> Is it needed? I think you can use `__atomic_store_n` or `__atomic_store`

@vinser52 AFAIK `__atomic_store_n` or `__atomic_store` work only on Linux

Reply:
> this is for win so InterlockedExchange64

@bratpiorka Done